README.md

```diff
@@ -115,7 +115,46 @@ module "cdn" {
 
 ### Generating ACM Certificate
 
-Use the AWS cli to [request new ACM certifiates](http://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request.html) (requires email validation)
+```hcl
+# acm can only be created in us-east-1
```

Comment:
Minor comment improvement suggestion - ACM certs can be in any region but for use in with CloudFront they indeed need do be in `us-east-1`